### ISSUE TITLE
Possibility to visualize more than 1 locus 

### ISSUE THREAD

**Issue by katerinaoleynikova** [Illumina/REViewer#10]
Good day,

I would like to get some information about whether it is possible to get the results not only for one locus, but for some of them. If not, is it appropriate to add, for instance, --loci {LocusID1, LocusID2, .. LocusIDN} in the future?

Thanks a lot.

**Comment by egor-dolzhenko** [comment]
Thank you for the suggestion! This is not possible at the moment, but we can certainly add this feature.

**Comment by katerinaoleynikova** [comment]
Glad to hear, thanks a lot!

**Comment by katerinaoleynikova** [comment]
Hii, want to disturb you once again,

When are you going to include this feature in the tool?

Thank you

**Comment by egor-dolzhenko** [comment]
Apologies for taking a long time with this! I will do my best to add this feature by the end of the week.

**Comment by egor-dolzhenko** [comment]
I just implemented a basic support for multiple loci. They can now be specified as a comma-separated list (without spaces) like so: `--locus ATXN3,C9ORF72`. Here is a Linux binary: [REViewer-v0.2.1-linux_x86_64.gz](https://github.com/Illumina/REViewer/files/5923718/REViewer-v0.2.1-linux_x86_64.gz)
 
Thank you for suggesting this change and please let me know if you encounter any issues.

**Comment by katerinaoleynikova** [comment]
Thank you so much. Is it possible to add it in the next release?

Best wishes.

**Comment by egor-dolzhenko** [comment]
Absolutely! This improvement will be included in the next release. Meanwhile, please let me know if you have any issues with the binary attached above.

**Comment by katerinaoleynikova** [comment]
Good day!

That works, thank you so much!
